ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/GroupParametersService.java

```diff
@@ -0,0 +1,17 @@
+package net.corda.v5.ledger.utxo;
+
+import net.corda.v5.membership.GroupParameters;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * {@link GroupParametersService} allows flows to access {@link GroupParameters} for further verification,
+ * for example, accessing to check if issued money is from authoritative party in flows.
+ */
+public interface GroupParametersService {
```

Comment:
LGTM, but my only thought here would be that a name like `GroupParametersLookup` would align better with `NotaryLookup` and `MemberLookup` services.

Reply:
wdyt? @lankydan

Reply:
I'm fine with it being named `GroupParametersLookup` if that makes things more consistent. @vlajos what do you think?

Reply:
I'm happy either way btw so if you prefer the Service suffix that's fine by me

Reply:
Lets go with `GroupParametersLookup` then. We can always change it later if we aren't happy with it.